pkg/features/kube_features.go

```diff
@@ -98,7 +94,6 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	ExternalTrafficLocalOnly:                    {Default: true, PreRelease: utilfeature.GA},
```

Comment:
How about the flag itself?  What happens to users who had the `--feature-gate=` flag set, using this value, and now this value is gone -- does the flag parsing fail, or do we just ignore it?